planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -311,9 +316,7 @@ TurnSignalInfo NormalLaneChange::get_terminal_turn_signal_info() const
 
   const double nearest_dist_threshold = common_param.ego_nearest_dist_threshold;
   const double nearest_yaw_threshold = common_param.ego_nearest_yaw_threshold;
-  const size_t current_nearest_seg_idx =
-    autoware::motion_utils::findFirstNearestSegmentIndexWithSoftConstraints(
-      path.points, current_pose, nearest_dist_threshold, nearest_yaw_threshold);
+  const size_t current_nearest_seg_idx = common_data_ptr_->transient_data.current_path_seg_idx;
```

Comment:
```suggestion
  const auto current_nearest_seg_idx = common_data_ptr_->transient_data.current_path_seg_idx;
```

if possible i want to standardize by using auto